lib/mapper.js

```diff
@@ -22,13 +52,52 @@ exports.track = function(track){
     objCase.del(props, 'revenue');
   }
 
-  return {
-    email: track.email(),
-    userId: track.userId(),
-    eventName: track.event(),
-    createdAt: unixTime(track.timestamp()),
-    dataFields: props
-  };
+  var eventName = track.event();
+  var cart = props["products"] || props["cart"];
```

Comment:
Sweet, one less thing to worry about. Sorry about these (probably silly) questions, I'm not at all familiar with your libraries.